extensions/all-extensions/pom.xml

```diff
@@ -2,8 +2,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>org.mule.extensions</groupId>
-        <artifactId>mule-extensions</artifactId>
+        <groupId>org.mule</groupId>
+        <artifactId>mule</artifactId>
```

Comment:
This groupId:artifactId already exists, my understanding is that it should be org.mule.extensions.mule-extensions